source/application/CLS/CLSAppController.cpp

```diff
@@ -60,6 +61,14 @@ bool CLSAppController::startup()
 	return false;
 }
 
+void CLSAppController::shutdown()
+{
+	if (userConfiguration_)
```

Comment:
It seems like the beamline-specific user configurations are all named something different (eg. bioxasUserConfiguration_, ideasUserConfiguration_, etc). Is there a case where a user configuration is just called userConfiguration_? It seems like this would be the only case where this code is executed.